tensorflow/contrib/cmake/README.md

```diff
@@ -165,6 +165,29 @@ Step-by-step Windows build
 
 3. Invoke CMake to create Visual Studio solution and project files.
 
+   **NOTE**: MSVC does not support [the flag `-march=native` defaulted on `CMakeLists.txt`](https://github.com/tensorflow/tensorflow/blob/master/tensorflow/contrib/cmake/CMakeLists.txt#L78) 
+   so in order to build TensorFlow with MSVC using advanced CPU instructions
+   you will need to edit `CMakeLists.txt` and explicitly define the options
+   as in `/arch:[ARG1|ARG2|ETC]`, e.g.:
```

Comment:
You are right, it turned out way too complex. That would be ideal, great suggestion